docs/samples/accelerators/tensorflow-gpu.yaml

```diff
@@ -7,7 +7,7 @@ metadata:
 spec:
   default:
     tensorflow:
-      modelUri: "gs://kfserving-samples/models/tensorflow/flowers"
+      storageUri: "gs://kfserving-samples/models/tensorflow/flowers"
```

Comment:
These examples are still using v1alpha1, storageUri is in v1alpha2. Let’s just all update them to v1alpha2?

Reply:
done!